pkg/cmd/pipelinerun/delete_test.go

```diff
@@ -358,6 +358,15 @@ func TestPipelineRunDelete(t *testing.T) {
 			wantError:   false,
 			want:        "Are you sure you want to delete all PipelineRuns in namespace \"ns\" (y/n): All PipelineRuns deleted in namespace \"ns\"\n",
 		},
+		{
+			name:        "Delete all by ignore-running",
```

Comment:
can you please add more test scenarios, like this behavior should be the same without flag also, and it is not deleting pipeline when setting the flag false? The different status you can try